
<launch>  
  <!-- vim: set ft=xml noet : -->
  <!-- base node launch file-->

  <!-- Use YOLOv3 -->
  <arg name="network_param_file" default="$(find darknet_ros)/config/yolov3-tiny.yaml"/>
  <arg name="image" default="/csi_cam_0/image_raw" />

  <!-- Include main launch file -->
  <include file="$(find darknet_ros)/launch/darknet_ros.launch">
    <arg name="network_param_file" value="$(arg network_param_file)"/>
    <arg name="image" value="$(arg image)" />
  </include>

</launch>
